openedx_proversity_reports/reports/last_page_accessed.py

```diff
@@ -57,13 +58,9 @@ def get_last_page_accessed_data(course_list):
             )
             parent_tree_name = ''
             vertical_block_id = ''
-            user_role = 'student'
 
             if last_completed_child_position:
-                user_course_role = course_access_role().objects.filter(
-                    user=user,
-                    course_id=course_key
-                )
+                user_course_role = get_user_role(user, course_key)
```

Comment:
This line is not necessary , you can call the method directly in the line 80

Reply:
Done.